tensorboard/webapp/plugins/npmi/views/data_selection/metric_search/metric_search_component.ts

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
[no action needed / note to self]: this component looks fairly generic, aside from the placeholder text.  It would be nice to have our own reusable version in TensorBoard, so that we don't have every dashboard re-implementing the same logic.

Making such a reusable component is outside of the scope of this PR, though.